src/SFA.DAS.EmployerIncentives.Api/Models/SendBankDetailsEmailRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+namespace SFA.DAS.EmployerIncentives.Api.Models
+{
+    public class SendBankDetailsEmailRequest
+    {
+        public long AccountId { get; set; }
+        public long AccountLegalEntityId { get; set; }
```

Comment:
I may well I have slightly misunderstood the purpose of this functionality, but don't we send the bank the LegalEntityId not the accountLegalEntityId. However I see it's for an email so it's probably correct

Reply:
It's for monitoring purposes, if we have a failed message then we know which account/account legal entity it's for